app/components/kickout-modal.hbs

```diff
@@ -1,18 +1,18 @@
 <div data-test-kickout-modal class='kickout-modal'>
-  <h4 data-test-kickout-heading>Do you want to remove {{@peerToRemove}}?</h4>
+  <h4 data-test-kickout-heading>{{@message}}</h4>
   <div data-test-kickout-buttons class='kickout-modal__buttons'>
     <Reusables::Button
-      @text='Cancel'
+      @text={{@closeText}}
       @variant='dark btn--sm'
       @test='kickout-button-cancel'
       @onClick={{this.closeModal}}
     />
     <Reusables::Button
-      @text='Remove'
+      @text={{@confirmText}}
       @variant='pink btn-dark btn--sm'
       @test='kickout-button-remove'
       @type='button'
-      @onClick={{this.removePeer}}
+      @onClick={{this.confirmFun}}
```

Comment:
confirmFun is not looking good in naming, what about `onConfirmClick` or just `onConfirm`